### PR TITLE
contributors

### PR DESCRIPTION
**Pull request by kristojorg**
This is a draft PR to resolve https://jira.nypl.org/browse/SIMPLY-3653 where contributors are not showing up right in list view or book details view.

### CODE REVIEW

File: src/components/BookList.tsx [hunk 1]
```diff
@@ -110,6 +110,18 @@ export const BookList: React.FC<{
   );
 };
 
+function getAuthorsString(book: AnyBook): string {
+  const { authors } = book;
+  if (!authors) return "Unknown Author";
+  const authorsArray = getAuthors(book, 2);
+
+  if (authors.length > 2) {
+    authorsArray.push(`& ${authors?.length - 2} more`);
```
Comment (kaplanmichael) [inline src/components/BookList.tsx:119]: Does `authors` need the question mark here?  If there aren't any authors that line of code I think won't be executed in the first place.

File: src/components/__tests__/BookList.test.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ test("truncates authors", () => {
   );
   const utils = render(<BookList books={[longBook]} />);
 
-  expect(utils.getByText("one, two & 3 more"));
+  expect(utils.getByText("one, two, & 3 more"));
```
Comment (kaplanmichael) [inline src/components/__tests__/BookList.test.tsx:53]: This Oxford comma just made my day.  This PR now gets 100 bonus points!

File: src/components/BookList.tsx [hunk 3]
```diff
@@ -119,7 +131,7 @@ export const BookListItem: React.FC<{
   const book = loanedBook ?? collectionBook;
 
   // uses contributors if there are no authors
```
Comment (kaplanmichael) [inline src/components/BookList.tsx:133]: Is this comment accurate? Looking at `getAuthorsString`, I didn't see anything related to contributors (it seems like that's all handled in the `parse` method.

**Review by kaplanmichael — CHANGES_REQUESTED** [review]
Looks good. Just a few quick questions:

File: src/components/BookList.tsx (hunk 1 above)
Comment (kristojorg) [reply src/components/BookList.tsx:119]: Good point thanks

File: src/components/__tests__/BookList.test.tsx (hunk 2 above)
Comment (kristojorg) [reply src/components/__tests__/BookList.test.tsx:53]: Hahah it was a necessary fix

File: src/components/BookList.tsx (hunk 3 above)
Comment (kristojorg) [reply src/components/BookList.tsx:133]: Yep it's accurate, it's withing the `getAuthors` method which is called inside `getAuthorsString`. I moved them next to each other in `utils/book.tsx` more obvious.

**Pull request merged**